packages/vite/schemas/manifest.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Vite Manifest",
+  "type": "object",
+  "patternProperties": {
+    "^.+$": {
```

Comment:
Is this too wide ? If so, does it really matter ?

Reply:
Given some input entrypoints can be virtual files, I'd say any string is valid 👍